app/models/schedule.rb

```diff
@@ -17,7 +17,7 @@ class Schedule < ApplicationRecord
 
   validates_presence_of :facility
 
-  scope :active, -> { where(id: Product.not_archived.with_schedule.select(:schedule_id)).order(:name) }
+  scope :active, -> { where(id: Instrument.not_archived.with_schedule.select(:schedule_id)).order(:name) }
```

Comment:
This is definitely supposed to be Instrument only? I think that makes sense with the schedule page, just checking though since it's been a minute since I did much in this code base.

Reply:
Absolutely — I also double checked, and there are 0 products that have a `schedule_id` and are not `Instruments` (what a complicated way to say the only products with schedules, are instruments)!